org.videolan.VLC.Plugin.fdkaac.json

```diff
@@ -0,0 +1,68 @@
+{
+  "id": "org.videolan.VLC.Plugin.fdkaac",
+  "build-extension": true,
+  "runtime": "org.videolan.VLC//stable",
+  "sdk": "org.freedesktop.Sdk//1.6",
+  "tags": ["free"],
+  "separate-locales": false,
+  "build-options": {
+    "cflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+    "cxxflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+    "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+    "env": {
+      "V": "1"
+    }
+  },
+  "modules": [
+    {
+      "name": "fdkaac",
+      "config-opts": [
+        "--with-pic",
+        "--disable-static",
+        "--enable-shared",
+        "--prefix=/app/share/vlc/extra/fdkaac"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/mstorsjo/fdk-aac/archive/v0.1.6.tar.gz",
+          "sha256": "adbcd793e406e1b88b3c1c41382d49f8c27371485b823c0fdab69c9124fd2ce3"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+    {
+      "name": "vlc",
+      "rm-configure": true,
+      "config-opts": [
+        "BUILDCC=/usr/bin/gcc -std=gnu99",
+        "PKG_CONFIG_PATH=/app/share/vlc/extra/fdkaac/lib/pkgconfig",
+        "--disable-a52",
+        "--disable-lua",
+        "--enable-fdkaac",
+        "--prefix=/var/tmp/install"
+      ],
+      "post-install": [
+        "mkdir -p /app/share/vlc/extra/fdkaac/plugins",
+        "mv /var/tmp/install/lib/vlc/plugins/codec/libfdkaac_plugin.so /app/share/vlc/extra/fdkaac/plugins"
```

Comment:
You can probably just pass the single plugin to `make-args` and avoid building the rest.

Reply:
I think I tried, I'll try again I am not that good in autotools machinery so I perhaps missed something.

Reply:
I played a bit with make and couldn't find a way to only build the plugin, sorry.